test/unit-test/dawn/Optimizer/Passes/samples/test_compute_maximum_extent_02.cpp

```diff
@@ -0,0 +1,35 @@
+//===--------------------------------------------------------------------------------*- C++ -*-===//
+//                         _       _
```

Comment:
why do we need these samples?

Reply:
this was to be able to regenerate the .sir references we use for the unittest. Do you remember?

Reply:
yeah, right